.storybook/config.js

```diff
@@ -0,0 +1,3 @@
+import { configure } from '@storybook/react';
+
+configure(require.context('../src', true, /\.stories\.js$/), module);

```

Comment:
Storybook utilizes configs in `.storybook/` folder (see https://github.com/taskcluster/material-ui-json-schema-viewer/issues/15#issuecomment-569051741). 
Overall, this `config.js` file helps Storybook to work 🌈 .

ℹ️  I've also tried customizing Storybook's webpack config to utilize Neutrino's webpack config, however, had come across some undesirable results.

- complexity in configuring Neutrino's webpack config to match Storybook's `full control mode` which returns a function instead of an object form (https://storybook.js.org/docs/configurations/custom-webpack-config/#webpack-customisation-modes)
- if Neutrino's webpack config gets hooked into Storybook's webpack config, linting applies the following error to Storybook's `stories` files: `JSX not allowed in files with extension '.js' (react/jsx-filename-extension)`
   - since `stories` must be in `.js` extenstion file formats (if in `jsx` extension, storybook UI does not display correctly), would need to add eslint config to disable that error

Since Storybook works fine even without hooking Neutrino's webpack config (and hooking Neutrino's webpack config would rather produce undesirable consequences), I think it's best we leave Storybook's config as-is.
Maybe if Storybook implements changes to use `storybook.config.js` to export a prest, we may be able to consider hooking webpack in the future :)

Reply:
Seems like webpack customization will be necessary after all! (see https://github.com/taskcluster/material-ui-json-schema-viewer/issues/15#issuecomment-569218385)

Reply:
Interesting! Thank you for the explanation, very helpful. I'll try to look into it soon and get back to you.